specifyweb/frontend/js_src/lib/components/FormEditor/viewSpec.ts

```diff
@@ -35,15 +35,15 @@ export const formDefinitionSpec = (table: SpecifyTable | undefined) =>
       syncers.xmlChild('enableRules', 'optional'),
       syncers.maybe(syncers.object(legacyBusinessRulesSpec()))
     ),
-    rows: rows(table),
+    definitions: definitions(table),
```

Comment:
renamed `rows` to `definitions` to be less ambiguous (there is an XML child named `rows` that contains all the rows for a given definition, and there could be multiple such definitions so `rows or rows` - having the prop be called `rows` was ambiguous as to which `rows` it's referring too)